index.html

```diff
@@ -5,39 +5,167 @@
+        </div>
+        <div class="grid-item">
+            <img class="work-section-image" src="ressources/Work-4.png" alt="Picture for the first work of Benjamin Patin">
+            <h2>Multi-Post Stories</h2>
+            <ul class="client-info">
+              <li class="client-name" id="info-client">CANOPY</li>
+              <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
+              <li class="project-style"id="info-client">Back End Dev</li>
+              <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
+              <li class="project-year"id="info-client">2015</li>
+            </ul>
+            <p class="project-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="language-used">
+              <div class="tags">html</div>
+              <div class="tags">css</div>
+              <div class="tags">javaScript</div>
+          </div>
+          <button type="button" class="button">See Project!</button>
+        </div>
+      </div>
 
-        <!-- Add font awesome icons -->
-        <ul class="icon-list">
-          <li><a href="https://facebook.com/benj.p009" target="_blank" rel="noopener" class="fa fa-facebook"></a>
-          <a href="https://instagram.com/benjamin_patin" target="_blank" rel="noopener" class="fa fa-instagram"></a>
-          <a href="https://linkedin.com/in/benjaminpatin" target="_blank" rel="noopener" class="fa fa-linkedin"></a>
-          <a href="https://twitter.com/benjp009" target="_blank" rel="noopener" class="fa fa-twitter"></a>
-          <a href="https://github.com/benjp009" target="_blank" rel="noopener" class="fa fa-github"></a></li>
-        </ul>
+      <!-- The "about me section is here"-->
+      <div class="about-me">
+        <h2> About Myself </h2>
+        <p> Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <div class="social">
+          <div class="let-connect">Let's Connect</div>
+          <ul class="icon-list">
+            <a href="https://facebook.com/benj.p009" target="_blank" rel="noopener" class="fa fa-facebook"></a>
+            <a href="https://instagram.com/benjamin_patin" target="_blank" rel="noopener" class="fa fa-instagram"></a>
+            <a href="https://linkedin.com/in/benjaminpatin" target="_blank" rel="noopener" class="fa fa-linkedin"></a>
+            <a href="https://twitter.com/benjp009" target="_blank" rel="noopener" class="fa fa-twitter"></a>
+            <a href="https://github.com/benjp009" target="_blank" rel="noopener" class="fa fa-github"></a></li>
+          </ul>
+        </div>
+        <button type="button" class="button">Get Resume</button>
+        <div class="self-category">
+          <hx>Languages</hx>
```

Comment:
- [ ] Let's fix these "hx" tags, these are not actually valid HTML tags, instead use `h2`...`h6`. Which one to use is up to you. Just remember that the closer they are to the `h1` tag, the important the element that they wrap.